src/views/scripts/tabs.blade.php

```diff
@@ -0,0 +1,60 @@
+<script>
+    const tabs = () => {
+        return {
+            active: 0,
+
+            activeTabClass: '',
+
+            inactiveTabClass: '',
+
+            init(defaultActive) {
+                this.setActiveTab(defaultActive)
+            },
+
+            tabTitles() {
+                return this.tabItems().map(child => child.dataset.title)
+            },
+
+            tabItems() {
+                return [].slice.call(this.$refs.tabItems.children)
+            },
+
+            focusActiveTab() {
+                [].slice.call(this.$refs.tabs.getElementsByTagName('button'))
+                    .find(node => node.id === `tab-${this.active}`)
+                    ?.focus()
+            },
+
+            setActiveTab(index) {
+                if (index < 0 || index > this.tabItems().length - 1) {
+                    return
+                }
+
+                this.active = index;
+
+                this.focusActiveTab()
+
+                this.tabItems().forEach(item => {
+                    item.classList.add("hidden");
+                })
+
+                this.tabItems()[this.active].classList.remove("hidden");
```

Comment:
I reviewed this one more time and it is not possible to use the `index` for `x-show` since the tab items are rendered via `slot` and not a `foreach`

![image](https://user-images.githubusercontent.com/5126648/108719033-13038100-74ed-11eb-8ce5-73d2d7a3db65.png)

If we wanted to use `x-show`, we would have to pass in an `index` prop that would defeat the purpose of the new API. What do you think? Or am I missing something?